pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -400,6 +413,10 @@ cdef class {{ message.full_name }}:
 
     cpdef void ClearField(self, field_name):
         """Clears the contents of a given field."""
+        self._clearfield(field_name)
+        self._Modified()
+
+    cdef void _clearfield(self, field_name):
```

Comment:
`cdef` here will swallow the `ValueError` that gets raised if `field_name` is invalid, I think.